software/control/core/core.py

```diff
@@ -1613,7 +1614,7 @@ def acquire_at_position(self, region_id, current_path, fov):
             print(f"Acquiring image: ID={file_ID}, Metadata={metadata}")
 
             # laser af characterization mode
-            if self.microscope.laserAutofocusController.characterization_mode:
+            if self.do_reflection_af and self.microscope.laserAutofocusController.characterization_mode:
```

Comment:
@Alpaca233 please revert the change here. We'll want to save laser af image even if we do not enable laser af (e.g. do a z stack).